tests/integration/init/schemas/schemas_test_data_setup.py

```diff
@@ -43,28 +43,6 @@ def setUpClass(cls):
         setup_partner_schema_data("partner-registry", schemas_client)
         setup_schema_data_for_pagination("test-pagination", schemas_client)
         setup_non_partner_schema_data("other-schema", schemas_client)
-        # WHEN the user follows interactive init prompts
```

Comment:
Why removing the inputs below?

Reply:
Answered here https://github.com/aws/aws-sam-cli/pull/6331#discussion_r1403662701